internal/exec/resolvable/resolvable.go

```diff
@@ -54,8 +56,14 @@ func (*Object) isResolvable() {}
 func (*List) isResolvable()   {}
 func (*Scalar) isResolvable() {}
 
-func ApplyResolver(s *schema.Schema, resolver interface{}) (*Schema, error) {
+// TODO figure out a better way to handle passed config
+// this approach avoids updating signature of many functions
+var useFieldResolvers bool
+
+func ApplyResolver(s *schema.Schema, resolver interface{}, useFieldRes bool) (*Schema, error) {
```

Comment:
Instead of adding global state, how about adding a field to `schema.Schema`? There's a pointer to it everywhere `useFieldResolvers` is read: both in this func and as a member of `resolvable.execBuilder` (other usages are in methods on execBuilder).

Reply:
fixed